chsdi/views/features.py

```diff
@@ -494,24 +494,19 @@ def _find(request):
 def _format_search_text(columnType, searchText):
```

Comment:
here you can perform a simple functional test. The goal here is to speed up queries by transforming a `string` into the approriate type.